www/templates/search.html

```diff
@@ -12,8 +12,8 @@
     <div class="item item-divider">
       Recent Searches
     </div>
-    <div class="item" ng-repeat="search in searches.slice(-5).reverse() track by $index" ng-click="$parent.$parent.searchText = search; getResults();">
-      {{search}}
+    <div class="item recent-search" ng-repeat="search in searches.slice(-5).reverse() track by $index" ng-click="$parent.$parent.searchText = search; getResults();">
+        <p class="gurmukhi">{{search}}</p>
```

Comment:
This class would have to be dynamic depending upon the language of the search, as you can also search in English.

Also, not sure what is the purpose of the `recent-search` class? Why is extra padding needed here?

Thanks for the pull request / seva! It's much appreciated :)